metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/package/templates/metron.j2

```diff
@@ -37,3 +37,5 @@ SECURITY_ENABLED={{security_enabled|lower}}
 {% endif %}
 {% if metron_keytab_path is defined %}METRON_SERVICE_KEYTAB="{{metron_keytab_path}}"
 {% endif %}
+KAFKA_SECURITY_PROTOCOL="{{kafka_security_protocol}}"
+PARSER_TOPOLOGY_OPTIONS="/home/metron/.storm/storm.config"
```

Comment:
I did not find any existing property that defines the `PARSER_TOPOLOGY_OPTIONS` value.  It is hard coded in one of the MPack Python scripts (`parser_commands.py`) as `' -e ~' + self.__params.metron_user + '/.storm/storm.config'`.

I thought that if I just put the exact value here, we are half way to making it configurable in Ambari, if we choose to do so.

I also tried to use `~${METRON_HOME}/.storm/storm.config`, but the `ProcessBuilder` does not do tilde expansion for user's home directories.